.github/workflows/release.yml

```diff
@@ -6,35 +6,92 @@ on:
     - '*'
 
 jobs:
-  release:
-    name: Python 3.x on ${{ matrix.os }}
-    runs-on: ${{ matrix.os }}
+  release-ubuntu:
+    name: Ubuntu release
+    runs-on: ubuntu-latest
     strategy:
       fail-fast: false
```

Comment:
I thikn fail-fast (and all of strategy) is no longer needed, fail-fast is only between elements of the matrix in a single job, I believe. Same for the other jobs.